TFproviderTest.yml

```diff
@@ -171,29 +171,30 @@ pipelines:
             - cat GNUmakefile | sed -e "s/darwin_amd64/linux_amd64/g" > GNUmakefile.tmp
             - cp GNUmakefile.tmp GNUmakefile && rm GNUmakefile.tmp
             - cat GNUmakefile
-            - cat sample.tf | sed -e "s/version =.*/version = \"${NEXT_PROVIDER_VERSION}\"/g" > sample.tf.tmp
-            - cp sample.tf.tmp sample.tf && rm sample.tf.tmp
-            - cat sample.tf
+            ## commented out before the first version released. Version is hardcoded to 0.0.1
```

Comment:
If we aren't updating the version in `sample.tf` then please delete these lines.

Reply:
Same with L165...

Reply:
we will update it with the first version will be released

Reply:
removing it, not it's done in the makefile, like in Projects and RT providers